.storybook/main.js

```diff
@@ -38,6 +38,40 @@ module.exports = {
             use: ["@svgr/webpack"],
         });
 
+        // Support Svelte
+        config.module.rules.push({
+            test: /\.svelte$/,
+            use: [
+                {
+                    loader: "svelte-loader",
+                    options: {
+                        compilerOptions: {
+                            dev: true,
+                        },
+                        emitCss: true,
+                        hotReload: true,
+                    },
+                },
+            ],
+        });
+
+        // TODO: make the mock work
```

Comment:
I was chatting with some Volto folks, and they pointed me to https://6.docs.plone.org/storybook/?path=/story/edit-widgets-object-browser--single-element. I don't know whether any of that can be reused here. Also sometimes Storybook components break, as we don't have a way to test them in CI.

For a fully working demo, visit https://volto.demo.plone.org/, login as `admin/admin`, edit the page, then insert an image block.